src/components/Menu/MenuBar.vue

```diff
@@ -215,22 +228,37 @@ export default {
 }
 </script>
 
-<style scoped lang="scss">
+<style lang="scss">
+	$color-white: white;
```

Comment:
Can we use CSS variables instead of sass variables?
It will prevent theme issues.